features/config.feature

```diff
@@ -534,3 +534,14 @@ require_once(ABSPATH . 'wp-settings.php');
       """
       http://example.com
       """
+
+  Scenario: If there is a wrong value defined for `DOMAIN_CURRENT_SITE` constant
+    Given a WP install
+
+    When I run `wp option get home`
+    Then STDOUT should contain:
```

Comment:
There's a semantic difference you need to apply.

When the test is expected to fail, you'll need to use `When I try` instead of `When I run`, and `Then STDERR` instead of `Then STDOUT` because error messages are written to `STDERR`